pyproject.toml

```diff
@@ -8,6 +8,7 @@ license = "BSD 3-Clause License"
 [tool.poetry.dependencies]
 python = "^3.9"
 pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
```

Comment:
I think it's "dev-dependencies", isn't it?

Reply:
Right, I moved `pytest` as well.